### ISSUE TITLE
[Accessibility: Error Identification] Required format info missing in New Custom Backup

### ISSUE THREAD

**Issue by atreyeemukhopadhyay** [kimchi-project/ginger#253]
while creating new custom backup ,paths included and excluded fields do not show the required format.
Error message is also not displayed.

![custom backup](https://cloud.githubusercontent.com/assets/13199397/13980476/d940d4d4-f104-11e5-86d3-bcebebfe63e3.jpg)
